external/fv3fit/tests/training/test_train.py

```diff
@@ -20,29 +21,42 @@
 
 
 # training functions that work on arbitrary datasets, can be used in generic tests below
-GENERAL_TRAINING_TYPES = [
+REGRESSION_TRAINING_TYPES = [
     "convolutional",
     "sklearn_random_forest",
     "precipitative",
     "dense",
     "transformed",
 ]
+
+# unlabeled models that can be tested on all model configuration tests, but not those
+# that actually depend on the quality of supervised learning that occurs
+NON_REGRESSION_TRAINING_TYPES = ["min_max_novelty_detector"]
+
 # training functions that have restrictions on the datasets they support,
 # cannot be used in generic tests below
 # you must write a separate file that specializes each of the tests
 # for models in this list
-SPECIAL_TRAINING_TYPES = []
+SPECIAL_TRAINING_TYPES = [
+    "min_max_novelty_detector",
+]
 
 
 # automatically test on every registered training class
-@pytest.fixture(params=GENERAL_TRAINING_TYPES)
+@pytest.fixture(params=REGRESSION_TRAINING_TYPES + NON_REGRESSION_TRAINING_TYPES)
 def model_type(request):
     return request.param
 
 
+# automatically test on every registered training class with labeled data
+@pytest.fixture(params=REGRESSION_TRAINING_TYPES)
+def labeled_model_type(request):
```

Comment:
Could you also update the name here and in references to `regression_model_type`?